qa/pull-tester/rpc-tests.py

```diff
@@ -75,6 +75,7 @@
     'wallet_doublespend.py',
     'wallet_import_export.py',
     'wallet_isfromme.py',
+    'wallet_orchard_change.py',
```

Comment:
This isn't actually change-address related, right?

Reply:
Ah yeah, that was my hypothesis at the time, but this was actually a persistence issue. But `wallet_orchard_persistence` is already used, by your test that was hypothesised to be a persistence issue, but was actually a scope selection issue. So... I'm not going to rename the tests in this PR; we can fix it later.

Reply:
Filed #5783.